src/test/java/com/artipie/asto/FileStorageTest.java

```diff
@@ -92,6 +93,18 @@ void saveOverwrites() throws Exception {
         );
     }
 
+    @Test
+    void saveBadContentDoesNotLeaveTrace() {
+        this.storage.save(
+            new Key.From("a/b/c/"),
+            new Content.From(Flowable.error(new IllegalStateException()))
+        ).exceptionally(ignored -> null).join();
```

Comment:
@olegmoz Let's check in `exceptionally` block that the exception is really raised that we didn't "swallowed" it

Reply:
@Vatavuk this is checked in `StorageSaveAndLoadTest.shouldFailToSaveErrorContent`. It is good practice to check only one thing in test, so I'd keep the code as it is

Reply:
@olegmoz ok, I didnt see that